com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/message/Messages.java

```diff
@@ -197,6 +197,88 @@
     public static final String DELETED_DATA_FOR_NON_EXISTING_USERS = "Deleted data for no-longer existing users.";
 
     // Progress messages
+
+    public static final String MTA_SCHEMA_VERSION_DETECTED_AS = "Detected MTA schema version: \"{0}\"";
+    public static final String TARGET_DETECTED = "Detected deploy target \"{0}\"";
+    public static final String NO_DEPLOYED_MTA_DETECTED = "No deployed MTA detected";
+    public static final String DEPLOYED_MTA_DETECTED_WITH_VERSION = "Detected deployed MTA with ID \"{0}\" and version \"{1}\"";
+    public static final String ASSUMED_LIVE_AND_IDLE_COLORS = "Assuming {0} is the live and {1} is the idle MTA color";
+    public static final String DEPLOYED_MTA_COLOR = "Deployed MTA color: {0}";
+    public static final String NEW_MTA_COLOR = "New MTA color: {0}";
+    public static final String DETECTED_NEW_MTA_VERSION = "Detected new MTA version: \"{0}\"";
+    public static final String DEPLOYED_MTA_VERSION = "Deployed MTA version: {0}";
+    public static final String CREATING_SUBSCRIPTION_FROM_0_MODULE_TO_1_RESOURCE = "Creating configuration change subscription from MTA module \"{0}\" to MTA resource \"{1}\"...";
+    public static final String ASSIGNING_URI = "Assigning URI \"{0}\" to application \"{1}\"...";
+    public static final String ADDING_DOMAIN = "Adding domain \"{0}\"...";
+    public static final String CREATING_SERVICE_FROM_MTA_RESOURCE = "Creating service \"{0}\" from MTA resource \"{1}\"...";
+    public static final String SERVICE_CREATED = "Service \"{0}\" created";

```

Comment:
Can you double-check if **PollServiceOperationsExecution.java** doesn't still print this message?

Reply:
It does. Fixed.